pkg/cmd/util/clientcmd/clientcmd.go

```diff
@@ -68,6 +71,13 @@ func (cfg *Config) bindEnv() {
 	if value, ok := util.GetEnv("BEARER_TOKEN"); ok && len(cfg.CommonConfig.BearerToken) == 0 {
 		cfg.CommonConfig.BearerToken = value
 	}
+	if value, ok := util.GetEnv("BEARER_TOKEN_FILE"); ok && len(cfg.CommonConfig.BearerToken) == 0 {
+		if tokenData, err := ioutil.ReadFile(value); err == nil {
+			cfg.CommonConfig.BearerToken = strings.TrimSpace(string(tokenData))
+		} else {
+			glog.Error(err)
```

Comment:
I guess I'm ok with returning an error, though I'd want the rest of the processing to continue